cmd/unpriv-setup.go

```diff
@@ -107,23 +107,32 @@ func addSpecificEntries(file string, name string) error {
 
 	}
 
+	// newuidmap (and thus lxc-usernsexec, or more generally liblxc) will
+	// complain if the current uid is in the subuid allocation. So if it
+	// is, let's just advance the subuid allocation another 65536 uids. we
+	// don't need to check if this overlaps again, since we know that
+	// maxAlloc was the highest existing allocation.
+	if maxAlloc < currentId && currentId < maxAlloc + 65536 {
```

Comment:
Does the first one need to be <= ? if currentId is 100000 and maxAlloc is 100000, we want it to skip.

Reply:
Yes, yes it does. Thanks :)